### ISSUE TITLE
Make a new release.

### ISSUE THREAD

**Issue by TimotheeJeannin** [lellolillili/rallymodemod#1]
Hi ! 

Thank you for the amazing work on this mod ! I absolutly love it ! 

I see in the github repo that quite a lot of content has been added but hasn't been released yet. 

Is it possible to make a release ? 

Love your work ! 

Have a nice day !

**Comment by lellolillili** [comment]
Yeah, it's my bad. I've been really busy with work and life in general.
I'll try to release ASAP. Thank you for the reminder!
